github/handlers/ExecuteBlockActionHandler.ts

```diff
@@ -1044,6 +1044,18 @@ export class ExecuteBlockActionHandler {
                     }
                     break;
                 }
+
+                case ModalsEnum.GITHUB_LOGIN_ACTION :{
+                    const {user, room} = context.getInteractionData();
+                    if(room){
+                        await storeInteractionRoomData(
+                            this.persistence,
+                            user.id,
+                            room.id
+                        );
+                    }
+                    break;
+                }
```

Comment:
@Nabhag8848 We are persisting this. But is this memory being freed up ?

Reply:
Yep. Right over here: https://github.com/RocketChat/Apps.Github22/blob/main/github/GithubApp.ts#L85
We were using the same mechanism of storing interaction roomId. 
- But before we stored the roomId during execution of command which lead to this bug.
- Now we are storing on interaction of button.